src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -67,8 +70,10 @@ public DiagnosticControllerBase(IServiceProvider services, IRuntimeContext<TReso
             this._searchService = (ISearchService)services.GetService(typeof(ISearchService));
             this._supportTopicService = (ISupportTopicService)services.GetService(typeof(ISupportTopicService));
             this._kustoMappingCacheService = (IKustoMappingsCacheService)services.GetService(typeof(IKustoMappingsCacheService));
+            this._translationCacheService = (ITranslationCacheService)services.GetService(typeof(ITranslationCacheService));
```

Comment:
It doesn't look like your using this in this class.

Reply:
Good catch! Updated